api/params.go

```diff
@@ -699,6 +699,7 @@ func (p *Params) SetDefaults(gitSource, gitOwner, gitName, appLabel, buildVersio
 			"173.245.48.0/20",
 			"188.114.96.0/20",
 			"190.93.240.0/20",
+			"104.24.0.0/14",
```

Comment:
@afarid you would also need to udpate the test cases to include this iprange as well

Reply:
Done!

Reply:
Is the build currently failing?

Reply:
PR updated, The build is successful now.